### ISSUE TITLE
App crashes

### ISSUE THREAD

**Issue by diskmanti** [kort/kort-native#50]
As soon as I upload an answer to a mission the app crashes.

I am on a pure android one device. Installed the app from google play store.

**Comment by tobiasff3200** [comment]
I had the same issue but only if GPS is disabled

**Comment by diskmanti** [comment]
@TobiasMie You are right. Thanks for the heads up. Can i consider this issued solved and close it?

**Comment by tobiasff3200** [comment]
I thing not, because the app crashes and you have to restart it. This is especially for new users a bad thing. If they try the app without GPS and it crashes they will leave a bad rank in the appstore and uninstall it. Another aspect is that I would often use it without location services in areas that I know. Maybe there could be a message that the user should ensure that he edit the right element.

**Comment by TobiasMie** [comment]
I thing not, because the app crashes and you have to restart it. This is especially for new users a bad thing. If they try the app without GPS and it crashes they will leave a bad rank in the appstore and uninstall it. Another aspect is that I would often use it without location services in areas that I know. Maybe there could be a message that the user should ensure that he edit the right element.

**Comment by and01** [comment]
I have labelled it as a bug, and will be fixed in the next release
